orchagent/orchdaemon.cpp

```diff
@@ -64,6 +68,8 @@ event_handle_t g_events_handle;
 #define DEFAULT_MAX_BULK_SIZE 1000
 size_t gMaxBulkSize = DEFAULT_MAX_BULK_SIZE;
 
+std::chrono::time_point<std::chrono::high_resolution_clock> OrchDaemon::m_lastHeartBeat = std::chrono::high_resolution_clock::now();
```

Comment:
>m_lastHeartBeat [](http://example.com/codeflow?start=72&length=15)

If not static, should it be initialized in ctor?

Reply:
Fixed, initialized in ctor